db/migrations/20230207085755_order_items.sql

```diff
@@ -0,0 +1,18 @@
+-- migrate:up
+CREATE TABLE order_items (
+    id INT NOT NULL AUTO_INCREMENT PRIMARY KEY,
+    user_id INT,
+    product_id INT,
+    order_id INT,
+    order_status_id INT,
+    quantity INT,
+    created_at TIMESTAMP NOT NULL DEFAULT CURRENT_TIMESTAMP,
+    updated_at TIMESTAMP NOT NULL DEFAULT CURRENT_TIMESTAMP ON UPDATE CURRENT_TIMESTAMP,
```

Comment:
updated_at 컬럼은 데이터가 최초 생성 될 때엔 특정 시간이 기입되지 않도록 아래와 같이 수정해주세요.

```suggestion
    updated_at TIMESTAMP NULL ON UPDATE CURRENT_TIMESTAMP,
```